eng/versioning/set_versions.py

```diff
@@ -14,19 +14,19 @@
 # and the current versions.
 #
 #
-#    python utilities/set_versions.py --ut [library|external_dependency|all] --bt [client|data|management] --bq <BuildQualifierString> --ar <artifactId>
+#    python eng\versioning\set_versions.py --ut [library|external_dependency|all] --bt [client|data|management] --bq <BuildQualifierString> --ar <artifactId>
 #
 # Use case: increment the version of a given artifact in the approprate version_[client|data|management].txt file
 #
-#    python eng/versioning/set_versions.py --bt [client|data|management] --increment-version --artifact-id <artifactId>
```

Comment:
nit: Generally I use forward slashes for these things because they work on both windows and linux but backslashes will only work on windows.

Reply:
Easy to change. Updated with the latest commit,.